content/en/docs/pipelines/sdk/sdk-overview.md

```diff
@@ -117,9 +116,11 @@ The Kubeflow Pipelines SDK includes the following packages:
     [experiment](/docs/pipelines/concepts/experiment/) and returns an
     experiment object.
   * `kfp.Client.run_pipeline` runs a pipeline and returns a run object.
-  * `kfp.Client.pipeline_uploads.upload_pipeline_version` uploads a local file to create a pipeline version. [Follow an example to learn more about creating a pipeline version](/docs/pipelines/tutorials/sdk-examples)
-
-* [`kfp.notebook`](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.notebook.html)
+  * `kfp.Client.create_run_from_pipeline_func` compiles a pipeline function and submits it
+    for execution on KFP.
```

Comment:
Do not abbreviate Kubeflow Pipelines to KFP. Please fix on this line and the next two lines.

KFP -> Kubeflow Pipelines

Reply:
Done. Also applied to other places.